src/stories/4-elements/typography/typography.stories.tsx

```diff
@@ -0,0 +1,297 @@
+
+export const Variants = Template("text-variants").bind({});
+Variants.args = {
+	uiType: "body-bl",
+	children: {
+		"heading-xxl": {
+			uiType: "heading-xxl",
+			children: LOREM_IPSUM("heading-xxl"),
+		},
+		"heading-xl": {
+			uiType: "heading-xl",
+			children: LOREM_IPSUM("heading-xl"),
+		},
+		"heading-lg": {
+			uiType: "heading-lg",
+			children: LOREM_IPSUM("heading-lg"),
+		},
+		"heading-md": {
+			uiType: "heading-md",
+			children: LOREM_IPSUM("heading-md"),
+		},
+		"heading-sm": {
+			uiType: "heading-sm",
+			children: LOREM_IPSUM("heading-sm"),
+		},
+		"heading-xs": {
+			uiType: "heading-xs",
+			children: LOREM_IPSUM("heading-xs"),
+		},
+		"body-md": {
+			uiType: "body-md",
+			children: LOREM_IPSUM("body-md"),
+		},
+		"body-sm": {
+			uiType: "body-sm",
+			children: LOREM_IPSUM("body-sm"),
+		},
+		"body-bl": {
+			uiType: "body-bl",
+			children: LOREM_IPSUM("body-bl"),
+		},
+		"body-xs": {
+			uiType: "body-xs",
+			children: LOREM_IPSUM("body-xs"),
+		},
+	},
+};
+
+export const Weights = Template("text-weights").bind({});
+Weights.args = {
+	uiType: "body-bl",
+	children: {
+		default: {
+			uiType: "body-bl",
+			children: LOREM_IPSUM("Default"),
+		},
+		bold: {
+			uiType: "body-bl",
+			children: LOREM_IPSUM("Bold"),
+			weight: "bold",
+		},
+		semibold: {
+			uiType: "body-bl",
+			children: LOREM_IPSUM("Semibold"),
+			weight: "semibold",
+		},
+		light: {
+			uiType: "body-bl",
+			children: LOREM_IPSUM("Light"),
+			weight: "light",
+		},
+	},
+};
+
+export const ArrayOfText = Template("text-array").bind({});
+ArrayOfText.args = {
+	uiType: "body-bl",
+	children: ["This", "is", "an", "array", "of", "text"],
+};
+
+export const ArrayOfStyledText = Template("text-array-plain-styled").bind({});
+ArrayOfStyledText.args = {
+	uiType: "body-bl",
+	children: ["This", "<mark>is</mark>", "<u>a</u>", "<strong>styled</strong>", "<i>sentence</i>"],
+};
+
+export const InlineText = Template("text-inline").bind({});
+InlineText.args = {
+	uiType: "body-bl",
+	children: {
+		"text-start": {
+			uiType: "body-bl",
+			children: "This is ",
+			inline: true,
+		},
+		"text-body": {
+			uiType: "body-bl",
+			children: "an inline ",
+			inline: true,
+		},
+		"text-end": {
+			uiType: "body-bl",
+			children: "text",
+			inline: true,
+		},
+	},
+};
+
+export const ParagraphText = Template("text-paragraph").bind({});
+ParagraphText.args = {
+	uiType: "body-bl",
+	children: {
+		"text-paragraph-one": {
+			uiType: "body-bl",
+			children: "This is the first paragraph",
+			paragraph: true,
+		},
+		"text-paragraph-two": {
+			uiType: "body-bl",
+			children: "This is the second paragraph",
+			paragraph: true,
+		},
+		"text-paragraph-three": {
+			uiType: "body-bl",
+			children: "This is the third paragraph",
+			paragraph: true,
+		},
+	},
+};
+
+export const HTMLString = Template("text-html-string").bind({});
+HTMLString.args = {
+	uiType: "body-bl",
+	children: "<p>his is a paragraph in a HTML string</p>",
```

Comment:
minor typo, taking the chance to fix it

```suggestion
	children: "<p>This is a paragraph in a HTML string</p>",
```